src/transformers/models/detr/image_processing_detr_fast.py

```diff
@@ -863,6 +863,7 @@ def preprocess(
             input_data_format = infer_channel_dimension_format(images[0])
         if input_data_format == ChannelDimension.LAST:
             images = [image.permute(2, 0, 1).contiguous() for image in images]
+            input_data_format = ChannelDimension.FIRST
```

Comment:
I guess this is a bug and here you fix it right?

Reply:
Yes it was, sorry for the lack of explanation. It's actually now fixed by another PR